src/systemd-status-mail

```diff
@@ -50,7 +52,16 @@ EOF
 	if [ -n "${RELAYHOST}" ]; then
 	    RELAY="-Ssmtp=${RELAYHOST}"
 	fi
-	mailx $MAILX_OPTIONS -Ssendwait -s "$SERVICE ($HOSTNAME)" -r "$FROM" "$RELAY" "$TO" <<EOF
+	if [ -n "${MAILX_AUTH_USER}" ]; then
+	    ENV="smtp-auth-user=${MAILX_AUTH_USER}"
+	fi
+	if [ -n "${MAILX_AUTH_PASSWORD}" ]; then
+	    ENV="$ENV smtp-auth-password=${MAILX_AUTH_PASSWORD}"
```

Comment:
Missing quoting, will break if there is anything unusual in the value

Reply:
Fixed by using `${MAILX_AUTH_PASSWORD@Q}`.